tests/test_api_groups_create.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from tests.helpers.api_utils import assert_response_status
+
+
+def test_create_group_without_hosts(api_create_group, db_get_group_by_name):
+    group_data = {"name": "my_awesome_group"}
```

Comment:
Is this valid? It's my understanding that you're supposed to pass the full object to a POST, which would lead me to expect something like this:
```suggestion
    group_data = {"name": "my_awesome_group", "host_ids": []}
```